map_merging/Cluster sourceMap
map_merging/Cluster mergedMap
map_merging/PairNumber[] matchList

